cymetric/timeseries.py

```diff
@@ -0,0 +1,321 @@
+import warning
+
+import pandas as pd
+import numpy as np
+
+try:
+    from pyne import data
+    import pyne.enrichment as enr
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
+
+
+import cymetric as cym
+from cymetric import tools
+
+from cymetric.filter import get_transaction_df
+from cymetric.filter import get_transaction_nuc_df
+from cymetric.filter import get_transaction_activity_df
+from cymetric.filter import get_transaction_decayheat_df
+from cymetric.filter import get_inventory_df
+from cymetric.filter import get_inventory_activity_df
+from cymetric.filter import get_inventory_decayheat_df
+
+
+def get_transaction_timeseries(evaler, senders=(), receivers=(), commodities=(), nucs=()):
+    """
+    Shape the reduced transation Data Frame into a simple time serie. Applying nuclides selection when required.
+
+    Parameters
+    ----------
+    evaler : evaler
+    senders :  of the sending facility
+    receivers :  of the receiving facility
+    commodities :  of the commodity exchanged
+    nucs :  of nuclide to select.
+    """
+
+    if len(nucs) != 0:
+        nucs = format_nuc(nucs)
+
+    df = get_transaction_nuc_df(
+        evaler, senders, receivers, commodities, nucs)
+
+    group_end = ['ReceiverPrototype', 'SenderPrototype', 'Time']
+    group_start = group_end + ['Mass']
+    df = df[group_start].groupby(group_end).sum()
+    df.reset_index(inplace=True)
+
+    df = df[['Time', 'Mass']].groupby(['Time']).sum()
+    df.reset_index(inplace=True)
+
+    time = evaler.eval('TimeList')
+    df = add_missing_time_step(df, time)
```

Comment:
`add_mission_time_step()` doesn't seem to be imported. Is there a test for this function?